docs/developing.md

```diff
@@ -262,24 +261,6 @@ NOTE: The cluster name and domain used to create the certificate must match the
 Example:
 `hiveutil create-cluster mycluster --serving-cert=$HOME/mycluster.crt --serving-cert-key=$HOME/mycluster.key`
 
```

Comment:
Should we keep a link to the new doc from here?

Reply:
I'll add a link in the top level README

Reply:
Ended up moving the whole thing into using-hive.md given it's size.